cfme/tests/configure/test_access_control.py

```diff
@@ -790,8 +790,19 @@ def test_user_change_password(appliance, request):
 
 
 # Tenant/Project test cases
+
+@pytest.fixture(scope="module")
```

Comment:
**REQUIRED:** These are not necessary as it's just a one liner in the test that will soon go away....with the updated Collections PR

Reply:
Updated.